packages/kit/src/runtime/app/images.js

```diff
@@ -0,0 +1,47 @@
+import { DEV } from 'esm-env';
+import { sizes, loader, domains } from '__sveltekit/images';
```

Comment:
I'd prefer not to tie it to SvelteKit, so I think it'd be nicer to create a singleton instance rather than using a virtual module.

Reply:
What do you mean by that? How else would you get the config into the app?

Reply:
I'm actually wondering if the app even needs the config. It looks like you could keep it contained purely within `adapter-vercel`?